container/root/tests/php-fpm/5.6.goss.yaml

```diff
@@ -97,6 +97,8 @@ file:
       - '/^newrelic.daemon.logfile = \"\/dev\/stdout\"/'
       - '/^newrelic.loglevel = \"warning\"/'
       - '/^newrelic.daemon.loglevel = \"warning\"/'
+      - '!/^newrelic.transaction_tracer.enabled = true/'
+      - '!/^newrelic.distributed_tracing_enabled = true/'
```

Comment:
Test that the default container does not have tracing enabled at build time.

Reply:
nitpick, but could we check for false instead? this could match QUITE a few things...

Reply:
@mannytoledo it also looks like there could be a `;` infront of it, optionally. need the test to reflect this

Reply:
Yes, I think I should do both. 

I need to check for the absence of `newrelic.distributed_tracing_enabled = true` because that line would enable the feature before a user intends it. The current check does that.

Then I should also test for the presence of the default commented line starting with `;` because we expect it to exist for the sed to work. The default behavior is the line commented and set to false. I can add that test as well.

Any other thoughts?

Reply:
i'll never argue with multiple tests!